### PR TITLE
Fixed microrepl.py bug in e880a8c

### PR DESCRIPTION
**Pull request by vlasovskikh**
Fixed: AttributeError: 'Out' object has no attribute 'buffer' bug introduced in e880a8c.

### CODE REVIEW

File: scripts/microrepl.py [hunk 1]
```diff
@@ -165,10 +165,12 @@ def main():
     port = sys.argv[1]
     print('Device path', port)
 
-    term = connect_miniterm(port)
+    
 
     if os.name == 'nt':
-        term.console = Windows10Console()
+        miniterm.console = Windows10Console()
```
Comment (vlasovskikh) [inline scripts/microrepl.py:171]: @jdjjm I don't get how `miniterm.console` is used after that. It seems to me like this statement has no effect on what happens next.

No code in `serial/tools/miniterm.py` actually uses the global (file-level) variable `console`. Instead, instances of the class `Miniterm` use their instance-level attribute `console` via `self.console`.

File: scripts/microrepl.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ class Windows10Console(miniterm.Console):
     }
 
     def __init__(self) -> None:
-        super().__init__()
+        super(miniterm.Console, self).__init__()
```
Comment (vlasovskikh) [inline scripts/microrepl.py:115]: @jdjjm Don't you skip [the logic in the body](https://github.com/pyserial/pyserial/blob/master/serial/tools/miniterm.py#L107) of `miniterm.Console.__init__` that way?

**Comment by jdjjm** [comment]
Yeah I realise now it was not properly implemented. I changed the parent class of the patched console to ConsoleBase just like PR [#351](https://github.com/pyserial/pyserial/pull/351) on pyserial. Also removed the arguments pass in the super call. This version is working for me as far as I can tell. Arrow keys and backspace are all working. Do you see an issue inheriting from ConsoleBase instead of Console?

**Comment by vlasovskikh** [comment]
@jdjjm The `Console` class for Windows (`if os.name == 'nt': ...`) defines a few things on top of its base class `ConsoleBase`. See [its source code](https://github.com/pyserial/pyserial/blob/master/serial/tools/miniterm.py#L104). At least it defines the encoding (the code page) for the terminal so non-ASCII characters are handled correctly (I presume). There is also logic for cleaning up allocated resources. I would prefer just to patch `Console` a bit as I did in #105 instead of throwing it away.

**Comment by jdjjm** [comment]
That makes sense, just wasn't sure why it was working with ConsoleBase still. Anyway, #105 seems to be working now so do you want to close this PR?

**Comment by vlasovskikh** [comment]
@jdjjm Yes, let's proceed with #105.

**Pull request closed without merging**